Magento_CustomerCustomAttributes/templates/customer/address/edit.phtml

```diff
@@ -19,7 +19,7 @@
                 <?= /* @escapeNotVerified */ $block->getNameBlockHtml() ?>
 
                 <div class="input dashboard__input">
-                    <?php 
+                    <?php
                         $getCompanyValidationClass = $this->helper('Magento\Customer\Helper\Address')
```

Comment:
will be better to declare new var for `$this->helper('Magento\Customer\Helper\Address')` since it used multiple times

Reply:
done